help/include/automatic-billing.md

```diff
@@ -7,14 +7,33 @@ With automatic billing, you automatically purchase a Zulip license for each user
 in your organization at the start of each billing period (month or year), and
 whenever you need additional licenses.
 
-- [Deactivating a user](/help/deactivate-or-reactivate-a-user) frees up their
-  license for reuse. No refunds are given for unused licenses you have
-  purchased. For example, if you start a billing period with 10 users,
-  deactivate 5 users, and then add 2 new users, you will be billed for 10
-  licenses for the entire billing period.
+- [Deactivating a user](/help/deactivate-or-reactivate-a-user) frees
+  up their license for reuse. No refunds are given for unused licenses
+  you have purchased.
 
-- If you have no free licenses when you **add a user**, you will be automatically
-  billed for an additional license. You will be charged only for the remaining
-  part of the billing period. For example, if you are on a $8/user/month monthly
-  plan, you will be billed $4 for a user added half way through the month. Each
-  month, you will see a single charge for all licenses added that month.
+- If you have no free licenses when you **add a user**, you will be
+  automatically billed for an additional license. You will be charged
+  a pro-rated rate only for the remaining part of the billing
+  period. Each month, you will see a single invoice covering all
+  licenses added that month.
```

Comment:
```suggestion
- If you have no free licenses when you **add a user**, you will be
  automatically billed for an additional license. The charge will be triggered
  at the of the billing month. You will be charged
  a pro-rated rate only for the remaining part of the billing
  period. Each month, you will see a single invoice covering all
  licenses added that month.
```

I feel this is missing a sentence in the middle of this sort ("The charge will be triggered
  at the of the billing month."). I guess this is already implied by the last sentence in this paragraph, but it wasn't super clear to me.